pom.xml

```diff
@@ -59,12 +59,12 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-sql-client</artifactId>
-        <version>${vertx.version}-FOLIO</version>
+        <version>${vertx.version}</version>
```

Comment:
Please revert to use the -FOLIO version that [RMB-752](https://issues.folio.org/browse/RMB-752) has released.